sdk/digitaltwins/digitaltwins/review/digitaltwins.api.md

```diff
@@ -0,0 +1,360 @@
+## API Report File for "@azure/digitaltwins"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+import { OperationOptions } from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { RequestOptionsBase } from '@azure/core-http';
+import { RestResponse } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface AzureDigitalTwinsAPIOptionalParams extends coreHttp.ServiceClientOptions {
+    $host?: string;
+    apiVersion?: string;
+    endpoint?: string;
+}
+
+// @public
+export interface DigitalTwinModelsAddOptionalParams extends coreHttp.OperationOptions {
+    models?: any[];
+}
+
+// @public
+export type DigitalTwinModelsAddResponse = ModelData[] & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: ModelData[];
+    };
+};
+
+// @public
+export type DigitalTwinModelsGetByIdResponse = ModelData & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: ModelData;
+    };
+};
+
+// @public
+export interface DigitalTwinModelsListOptionalParams extends coreHttp.OperationOptions {
+    dependenciesFor?: string[];
+    digitalTwinModelsListOptions?: DigitalTwinModelsListOptions;
+    includeModelDefinition?: boolean;
+}
+
+// @public
+export interface DigitalTwinModelsListOptions {
+    maxItemCount?: number;
+}
+
+// @public
+export type DigitalTwinModelsListResponse = PagedModelDataCollection & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: PagedModelDataCollection;
+    };
+};
+
+// @public
+export interface DigitalTwinsAddHeaders {
+    etag?: string;
+}
+
+// @public
+export interface DigitalTwinsAddRelationshipHeaders {
+    etag?: string;
+}
+
+// @public
+export interface DigitalTwinsAddRelationshipOptionalParams extends coreHttp.OperationOptions {
+    relationship?: any;
+}
+
+// @public
+export type DigitalTwinsAddRelationshipResponse = DigitalTwinsAddRelationshipHeaders & {
+    body: any;
```

Comment:
The `any`-typed `body` property on main return types can be removed here and other places its used.

Reply:
The body has to be there and it is a user defined type. The "any" type makes sense here.

---
In reply to: [473242079](https://github.com/Azure/azure-sdk-for-js/pull/10560#discussion_r473242079) [](ancestors = 473242079)